cinn/pybind/frontend.cc

```diff
@@ -472,6 +473,53 @@ void BindFrontend(pybind11::module *m) {
       .def("reverse", &CinnBuilder::Reverse, py::arg("operand"), py::arg("axis"))
       .def("__str__", [](CinnBuilder &self) { return self.name(); });
 
+  auto computation = py::class_<CinnComputation, std::shared_ptr<CinnComputation>>(*m, "Computation");
+  py::class_<CinnComputation::CompileOptions>(computation, "CompileOptions")
+      .def_readwrite("use_decomposer", &CinnComputation::CompileOptions::use_decomposer)
```

Comment:
The underlying passes should be hidden under high-level APIs.
e.g. `use_ecomposer`, `do_prerun` is currently used only in either training mode or inference mode, so we might introduce some high-level API such `set_training_or_inference` latter.